### DISCUSSION TITLE
Fatal error JsonMachine\JsonMachine not found

### DISCUSSION THREAD

**Discussion by byCihat** [halaxa/json-machine#76] [Q&A]
hi 
their installations but i am getting error like this

Fatal error: Uncaught Error: Class 'JsonMachine\JsonMachine' not found in C:\xampp\htdocs\...

\JsonMachine\Items no probles

**Comment by halaxa** [comment]
Hello. Please read the release notes for versions `0.8.0` and `1.0.0` in CHANGELOG.md. It's explained there.

**Reply by byCihat** [reply]
` composer require halaxa/json-machine `

this way I installed it with the help of composer on the cmd side

How do I install for the latest version?

**Reply by halaxa** [reply]
The latest stable version might be installed like so `composer require halaxa/json-machine ^1.0`. Caret (`^`) makes sure that you will get the latest functions of major version `1` on each `composer update` without breaking changes, as it won't install version `2` when it comes out. See https://getcomposer.org/doc/articles/versions.md#caret-version-range-
